eng/common/TestResources/README.md

```diff
@@ -89,6 +89,12 @@ Remove-AzADServicePrincipal -ApplicationId $sp.ApplicationId -Force
 
 If you persisted environment variables, you should also remove those as well.
 
+Some scripts accept the `AdditionalParameters` parameter to control additional resource configuration options. For example:
```

Comment:
Glad this is being specified, though I'm not sure of the wording. What does "some scripts" refer to in this case?

I think you could be more specific that these parameters end up becoming parameters to the ARM template.

Reply:
Good call out.